cluster-stamp.json

```diff
@@ -274,12 +315,12 @@
                     "name": "Microsoft.Insights/default",
                     "type": "providers/diagnosticSettings",
                     "apiVersion": "2017-05-01-preview",
+                    "location": "[parameters('logAnalyticsWorkspaceLocation')]",
```

Comment:
re-validate for this resource type whether location is required or not.

Reply:
Done | Addressed from [06c29](https://github.com/mspnp/aks-secure-baseline/pull/158/commits/06c293627751aeadccc87849fa55cbb2e4434444)

diagnostic does not require the location as discussed while alerts will fail with being on the same location as the centralized LA

```json
"error": {
    "code": "Workspace not found. Please note that this may be due to alert rule location, which must be the same as the workspace location.",
    "message": {
      "lang": "en-US",
      "value": "Workspace not found. Please note that this may be due to alert rule location, which must be the same as the workspace location."
    }
  }
```

Reply:
Although above error is true, the new way of making `scheduledQueryRules` allows you to create this without specifying the LA resource id and with that we can avoid passing the location. 

Rosolved via https://github.com/mspnp/aks-secure-baseline/pull/158#discussion_r583693391